src/actions/player/playerTrendsActions.js

```diff
@@ -63,9 +68,22 @@ export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch)
           start_time: match.start_time,
           win,
         });
-
         return dataList;
       }, []);
+
+      // Apply mean-smoothing by taking differences of the previously computed integral
+      for (let i = 0; i < trends.length; i += 1) {
+        if (i < trends.length - chunkSize) {
+          trends[i].value = (trends[i + chunkSize].value - trends[i].value) / chunkSize;
```

Comment:
Instead of assigning to the `i`th element I think it should be the `i + chunkSize` element since we want each data point to reflect the last 20 games.